src/com/facebook/buck/android/DefaultAndroidManifestReader.java

```diff
@@ -47,7 +47,7 @@
    * to show up in the launcher.
    */
   private static final String XPATH_LAUNCHER_ACTIVITIES =
-      "/manifest/application/*[self::activity or self::activity-alias]" +
+      "/manifest/application/*[self::activity or self::activity-alias[not(@android:enabled) or @android:enabled='true']]" +
```

Comment:
This line is over 100 characters (can be spotted with `ant lint`).  Can you please fix that?